build.gradle.kts

```diff
@@ -75,58 +71,6 @@ subprojects {
         }
     }
 
-    val kotlinApiVersion = org.jetbrains.kotlin.gradle.dsl.KotlinVersion.KOTLIN_1_6
-    subproject.tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile>().configureEach {
-        if (!name.lowercase().contains("test")) {
-            this.compilerOptions {
-                jvmTarget.set(JvmTarget.JVM_11)
-                freeCompilerArgs.addAll(listOf("-Xjvm-default=all-compatibility", "-Xexpect-actual-classes"))
-                apiVersion.set(kotlinApiVersion)
-            }
-        }
-    }
-    subproject.tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinJvmCompile>().configureEach {
-        if (!name.lowercase().contains("test")) {
-            this.compilerOptions {
-                jvmTarget.set(JvmTarget.JVM_11)
-                freeCompilerArgs.addAll(listOf("-Xjvm-default=all-compatibility"))
-                apiVersion.set(kotlinApiVersion)
-            }
-        }
-    }
-
-    subproject.plugins.withType<JavaPlugin> {
```

Comment:
The configuration `subproject.plugins.withType<JavaPlugin> { ... }` was not put into `modelix-kotlin-api-version.gradle.kts` and was removed from here.

Is setting `sourceCompatibility` and `targetCompatibility` not needed?

Reply:
I moved this to the `java` block but i accidentally removed those changes. good catch

Reply:
We can actually omit these, as the toolchain takes priority. see [here](https://docs.gradle.org/current/javadoc/org/gradle/api/plugins/JavaPluginExtension.html#setSourceCompatibility(org.gradle.api.JavaVersion))